ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
No, I meant both the table AND the code :)

> If we remove the model from the code, then whether you import it from the registry or directly, the import will still fail, won't it?

I don't think so.
Try removing the code, then run a makemigrations, and you should have a DeleteModel migration that removes it from the registry (and delete the table, and I think remove the ContentType). But this model still exists in the registry for the steps *before* this given migration AFAIK.
It should be just straightforward.